### PR TITLE
General share button

### LINKED ISSUE #115: Share LiveGuide button
General purpose share button to share the app's URL. Let's implement on all 3 versions of the app.

It would be great if on mobile it can kick to the OS share menu.

### PR DESCRIPTION
**Pull request by jjandoc**
Fixes #115.

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -15,5 +16,9 @@
   GTM_ID = "GTM-NZ5LXRN"
   SHEETS_URL = "https://spreadsheets.google.com/feeds/list/1D5JBmEg1teTHy43Gu0lJqlTJCp_T3lmHljAO5l_tEwk/1/public/full?alt=json"
 
+[context.master.environment]
+  SHARE_URL = "https://www.boston.com/liveguide"
```
Comment (mclaros) [inline netlify.toml:20]: is there any consequence that the bostonglobe URL above has a trailing slash whereas this one doesnt?

File: util/createPopupWindow.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ * @param {Number} height - Height of the window in pixels.
+ */
+function createPopupWindow(url, name = 'share', width = 550, height = 420) {
+  if (process.server) {
+    return;
+  }
+  const windowProperties = {
+    width,
+    height,
+    left: Math.round(screen.width / 2 - width / 2),
```
Comment (mclaros) [inline util/createPopupWindow.js:16]: I feel dumb. I never realized there's a `screen` object now

Comment (jjandoc) [reply util/createPopupWindow.js:16]: I literally copied this script from Joymode, which I had copied + adapted from probably stack overflow. Same with the `serializeObject` utility.

File: netlify.toml (hunk 1 above)
Comment (jjandoc) [reply netlify.toml:20]: not really any consequence but let me look at the canonical url's for each of these and make sure they match just so I can justify either way.

**Comment by jjandoc** [comment]
Thanks @mclaros! Also tagging in @mstem here for final approval. Happy to get a code review if you want, but mostly looking for product approval. You can click the "Details" link next to "Deploy preview ready" above.

Screen Shot 2020-06-21 at 3 20 38 PM

**Pull request merged**